src/cast_core/src/CrudActor.cpp

```diff
@@ -1455,6 +1455,20 @@ struct CrudActor::PhaseConfig {
         // Node is convertible to bool but only explicitly.
         const bool perPhaseMetrics = bool(phaseContext["MetricsName"]);
         auto opCreator = op->second;
+        // Special case if we have set MetricsName at the phase level, and OperationMetricsName on
+        // the operation.
+        if (auto metricsName = phaseContext["MetricsName"].maybe<std::string>())
```

Comment:
Please put below block in braces

Reply:
This _may_ be clearer as:

```c++
if (auto metricsName = phaseContext["MetricsName"].maybe<std::string>();
     metricsName && metricsName != opName) {
    
}
```

Reply:
Also..wait..we have `metricName` (singular) and `metricsName` (plural)? That's just waiting for a typo or mis-read.

Reply:
I tried your syntax. I wasn't sure how to do it :) 
Also, good call on metricName/metricsName. Updated. My mind really wanted it to be singular, but it's plural in all existing places.